### PR TITLE
update ug according to suggestions

### LINKED ISSUE #223: [PE-D] User Guide: Format
Just some nits to pick for the formatting of the User guide: 

1. Some of the headers are not well formatted, you can see '##<Title>' on the markdown file. It was probably supposed to be a h2 but was incorrectly formatted.

2. The images are a little inconsistent. The images in the commands are of different window sizes, which makes the UG look slightly messy!

3. Glossary not updated

4. Set-up section could be slightly clearer, as some users may not understand jargon such as 'command prompt' or 'terminal'.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: sogggy/ped#9

### PR DESCRIPTION
**Pull request by GabriellaTeh**
Fixes #223 
Todo:
Change screenshots to be standardised after ui has been changed.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ PlaNus is a **task managing desktop application** for students in NUS with many
 Objectives of PlaNus:
 1. Provide students with a detailed breakdown of how much time they spend on their tasks and lessons in their respective modules.
 2. Allow students to have a calendar view of their schedule.
-3. Allocate tasks and lessons to their schedule automatically. (coming soon in v1.4)
+3. Allocate tasks and lessons to their schedule automatically. (Coming soon in v1.4)
```
Comment (BobbyZhouZijian) [inline docs/UserGuide.md]: I think we remove this since we are not implementing this feature anymore.

**Review by BobbyZhouZijian — COMMENTED** [review]
Have you fixed the `###<Title>` problem as mentioned in the issue?

**Comment by GabriellaTeh** [comment]
> Have you fixed the `###<Title>` problem as mentioned in the issue?

I could not find the problem.

**Review by BobbyZhouZijian — APPROVED** [review]
LGTM

**Pull request merged**